lib/Jenkins/API.pm

```diff
@@ -547,31 +553,25 @@ sub general_call
 
 =head2 response_code
 
-This method returns the HTTP response code from our last request to 
+This method returns the HTTP response code from our last request to
 the Jenkins server.  This may be useful when an error occurred.
 
 =cut
 
-sub response_code
-{
-    my $self = shift;
-    return $self->_client->responseCode;
-}
-
 =head2 response_content
 
-This method returns the content of the HTTP response from our 
-last request to the Jenkins server.  This may be useful when 
+This method returns the content of the HTTP response from our
+last request to the Jenkins server.  This may be useful when
 an error occurs.
 
 =cut
 
-sub response_content
-{
-    my $self = shift;
-    return $self->_client->responseContent;
-}
+=head2 response_header
+
+This method returns the specified header of the HTTP response from our
+last request to the Jenkins server.
```

Comment:
it would be cool if we could add an example of how this is used.  Your example of grabbing the location header would be particularly useful.

Reply:
I wanted to add an example but I was not sure about the pod style that you have followed?!

Reply:
Sticking the code indented normally formats it right, and add words to explain as necessary.

Reply:
Ok, how should we proceed then technically. Do you merge and add me as contributor and I will add a response_header example as an additional commit to colinnewell:master directly before you create the release?

Reply:
You can add more commits to this PR until I merge.  After that you create a new PR if you still have more things to add.  Feel free to add to this PR, and if you can't, we can do another later.